src/vs/workbench/common/editor.ts

```diff
@@ -383,6 +383,8 @@ export interface IEditorInput extends IDisposable {
 	 */
 	readonly resource: URI | undefined;
 
+	ariaLabel: string;
```

Comment:
Just for consistency I would call it `getAriaLabel` because all the other methods for label, description etc are methods too

Reply:
Thanks, tackled now.